src/simparams.jl

```diff
@@ -91,7 +91,7 @@ function load_params(rng=MersenneTwister(0);
   dist_death_time = LogNormal(2.610727719719777, 0.44476420066780653)
```

Comment:
```suggestion
  dist_death_time = LogNormal(1.6968258851693163, 1.205)
```

Reply:
why the second argument has such low precision? :)